rest/tsig_key.go

```diff
@@ -136,6 +138,13 @@ func (s *TsigService) Delete(name string) (*http.Response, error) {
 }
 
 var (
+	// Status Codes used
+	BadRequestCode          = 400
+	NotFoundStatusCode      = 404
+	AlreadyExistsStatusCode = 409
+
+	// ErrTsigBadRequest
+	ErrTsigBadRequest = errors.New("bad request error, expected in S1 (18/10/2021)")
```

Comment:
Let's just return the standard error for a bad request, we don't need to expose this.